spec.md

```diff
@@ -2315,6 +2421,10 @@ message NodeServiceCapability {
       GET_VOLUME_STATS = 2;
       // See VolumeExpansion for details.
       EXPAND_VOLUME = 3;
+      // If Plugin implements VOLUME_CONDITION capability
+      // then it MUST implement NodeGetVolumeStats RPC
+      // call for fetching volume condition.
```

Comment:
Same as controller capability, capability description should be a generic description (that way in the future if we add additional methods or fields that use this capability, this comment does not need to be updated).

```suggestion
      // Indicates the SP supports node level volume condition.
      // An SP MAY implement `VolumeCondition` in only the Node
      // Plugin, only the Controller Plugin, or both.
      // If `VolumeCondition` is implemented in both the Node and
      // Controller Plugins, it SHALL report from different perspectives.
      // If for some reason Node and Controller Plugins report
      // misaligned volume conditions, CO SHALL assume the worst case
      // is the truth.
      // Note that, for alpha, `VolumeCondition` is intended to be
      // informative for humans only, not for automation.
```

Reply:
Done